spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/features/logging.adoc

```diff
@@ -450,6 +450,7 @@ Spring Boot supports structured logging and has support for the following JSON f
 
 To enable structured logging, set the property configprop:logging.structured.format.console[] (for console output) or configprop:logging.structured.format.file[] (for file output) to the id of the format you want to use.
 
+WARNING: xref:#features.logging.custom-log-configuration[Custom Log Configuration] take precedence over structured logging.
```

Comment:
Thanks, @quaff. I wonder if people might miss this? They could read the paragraph about and then stop, happy that they know which configuration property to set. I think it might be more noticeable if we changed the paragraph above instead to start with something like "To enable structured logging, ensure that you aren't using custom log configuration and set the property…". WDYT?

Reply:
I agree.

Reply:
Actually, "Structured Logging" and "Custom Log Configuration" are not mutually exclusive, I updated the document. @wilkinsona